docs/deploy/security_groups.md

```diff
@@ -0,0 +1,31 @@
+The  AWS Load Balancer Controller manages security groups of two categories: frontend and backend.
+
+1. Frontend SG
+2. Backend SG
+
+## Frontend Security Groups
+
+Frontend security groups control which clients can access the load balancers. The frontend security groups can be configured with the `alb.ingress.kubernetes.io/security-groups` annotation on the Ingress resources. If the annotation is not specified, the controller will create one security group per load balancer, allowing traffic from `inbound-cidrs` to `listen-ports`.
+
+## Backend Security Groups
+
+A single backend security group controls the traffic between load balancers and their target EC2 instances and ENIs. This security group is attached to the load balancers and is used as the traffic source in the ENI/Instance security group rules. The backend security group is shared between multiple load balancers.
```

Comment:
EC2 instances or ENI, not and
EC2 instances for instance target, ENI for IP targets. Oftentimes, they share the same SG.